scripts/teams-clone-team/README.md

```diff
@@ -19,71 +22,131 @@ Excelsior, hum? :P
+        $message = $message.Trim()
+        Write-Host ($message)
+        if ($Logfile.Length -gt 0) {
+            Add-Content -Path $Logfile -Value $message -Force
+        }
+    }
+    
+    Function Get-FoldersFiles($folderUrl, [ref] $outAssets) {
+        $items = Get-PnPFolderItem -FolderSiteRelativeUrl $folderUrl -ItemType All    
+        # Loop through the folders  
+        foreach ($asset in $items) { 
+            # dont collect forms document library   
+            if ($asset.Name -ne "Forms") {               
+                $newItemURL = $folderUrl + "/" + $asset.Name   
+                $item = New-Object PSObject
+                $item | Add-Member -MemberType NoteProperty -Name 'ServerRelativeUrl' -Value $asset.ServerRelativeUrl
+                $item | Add-Member -MemberType NoteProperty -Name 'LocalPath' -Value ($newItemURL -replace "/", "\")
+               
+                $item | Add-Member -MemberType NoteProperty -Name 'IsFolder' -Value  ($null -ne $asset.ItemCount)
+                $item | Add-Member -MemberType NoteProperty -Name 'Leaf' -Value  ($newItemURL | Split-Path -Leaf  )
+                
+                $item | Add-Member -MemberType NoteProperty -Name 'RemotePath' -Value ($newItemURL | Split-Path) #(($newItemURL | Split-Path) -replace "\\", "/")
+                [Hashtable] $t = $outAssets.Value
+                $t.Add($asset.ServerRelativeUrl, $item)
+                if ($null -ne $asset.ItemCount) {
+                    Write-Log  "   Collecting  $newItemURL"
+                    Get-FoldersFiles -folderUrl $newItemURL -outAssets ([ref]$t)
+                }  
+            }
+        }
+    } 
+    Function Add-Content([string] $source , [string] $destination) {
+
+        Write-Log "    Start"
+        Write-Log "    Content from $source will be cloned to $destination"
+        ## Get all document libraries
+        Connect-PnPOnline -Url $source -Interactive
+        $doclib = Get-PnPList -Includes RootFolder | where-object { $_.BaseTemplate -eq 101 -and $_.Title -eq "Documents" }
+        $folderDest = ($destination + "/" + (Get-PnPProperty -ClientObject $doclib.RootFolder -Property Name))
+        $subfolders = Get-PnPFolder -List $doclib 
+        $subfolders.foreach({
+                Write-Log ("     $($_.ServerRelativeUrl) cloned to $folderDest")  
+                Copy-PnPFile -SourceUrl $_.ServerRelativeUrl -TargetUrl $folderDest -OverwriteIfAlreadyExists -Force   
+            })
+        Write-Log "    All content was cloned ! "
+        Write-Log "    End"
+    } 
+ 
 
-    $msg = "`n`r
+    $env:functionName = ""
+    $env:MainFunctionName = ""
+    $msg = "`n`r`n`r
 
-    █▀█ █▄░█ █▀█ █▀▀ ▄▀█ █▄░█ █▀▄ █▄█
-    █▀▀ █░▀█ █▀▀ █▄▄ █▀█ █░▀█ █▄▀ ░█░  `n    MSTeam Builder  `n`n    ...aka ... [team-clone-team]
+       █▀█ █▄░█ █▀█ █▀▀ ▄▀█ █▄░█ █▀▄ █▄█ 
+█▀▀ █░▀█ █▀▀ █▄▄ █▀█ █░▀█ █▄▀ ░█░  `nMSTeam Builder  `n`n    ...aka ... [team-clone-team]
```

Comment:
@ScoutmanPt this isnt rendering think might have some tabs missing, can you validate?

Reply:
Will do!

Reply:
@pkbullock , whats the best way to address this ? Usually I cancel the pull request and create a new one.

Reply:
Anyway, i've updated the branch on my side and the pull request is updated. Can you check? @pkbullock

Reply:
@pkbullock